src/blackdoor/cqbe/settings/Config.java

```diff
@@ -122,6 +121,73 @@ private void write(File out, JSONObject jo) throws ConfigurationException {
 		} catch (IOException e) {
 			throw new SettingSaveException();
 		}
+	}
 
+	private static String standardizeKey(String key) {
+		return key.toLowerCase().replace(' ', '_');
+	}
+
+	/**
+	 * Allows for settings to be grabbed from a config file without
+	 * instantiating a Configuration object.
+	 * <p>
+	 * 
+	 * @author Cj Buresch
+	 * @version 0.0.1 -- Feb 21, 2015
+	 */
+	public static class ConfigReadOnly {
+
+		/**
+		 * Returns a single setting from a setting file.
+		 * <p>
+		 * Primarily for people who are feeling super lazy. Does a couple
+		 * checks, but in the event that things go wrong, it will throw a
+		 * configuration exception.
+		 * 
+		 * @param filename
+		 * @param setting
+		 * @return
+		 * @throws ConfigurationException
+		 */
+		public static Object get(String filename, String setting)
+				throws ConfigurationException {
+			File file = new File(filename);
+			return get(file, setting);
+		}
+
+		/**
+		 * Returns a single setting from a setting file.
+		 * <p>
+		 * For people who are feeling less lazy and already have a file.
+		 * 
+		 * @param file
+		 * @param setting
+		 * @return
+		 * @throws ConfigurationException
+		 */
+		public static Object get(File file, String setting)
```

Comment:
everywhere else in java key value parameters are (key, value), let's not change that.